messaging/src/main/java/ids/messaging/protocol/http/IdsHttpService.java

```diff
@@ -416,16 +437,16 @@ private Response sendRequest(final Request request,
     }
 
     /**
-     * Logs the request message to be send in debug-level log.
+     * Logs the request message to be send in log.
      *
      * @param request The request message to be send.
      * @throws IOException If Buffer could not be accessed.
      */
-    private void debugLogRequest(final Request request) throws IOException {
+    private void logRequest(final Request request) throws IOException {
         final var requestCopy = request.newBuilder().build();
         final var buffer = new Buffer();
         requestCopy.body().writeTo(buffer);
-        log.debug("Sending request message: [code=(IMSMED0135), request=({})]", buffer.readUtf8());
+        log.info("Sending request message: {}", buffer.readUtf8());
```

Comment:
It seems, that there is already a debug message for built messages. I receive now following two messages:
```
2022-05-25T15:21:25,928 [https-jsse-nio-8080-exec-3] INFO - Using cached DAPS DAT. [expiration=(2022-05-25T16:21:24.000+0200)]
2022-05-25T15:21:25,959 [https-jsse-nio-8080-exec-3] DEBUG - Built request message. [header=({\r\n  "@context" : {\r\n    "ids" : "https://w3id.org/idsa/core/",\r\n    "idsc" : "https://w3id.org/idsa/code/"\r\n  },\r\n  "@type ...
2022-05-25T15:21:25,961 [https-jsse-nio-8080-exec-3] INFO - Sending request message: --2d500413-ed10-42c3-9c4a-7033d176ee0e\r\nContent-Disposition: form-data; name="header"\r\nContent-Length: 2580\r\n\r\n{\r\n  "@context" : ...
2022-05-25T15:21:25,961 [https-jsse-nio-8080-exec-3] INFO - Sending request to https://localhost:8080/api/ids/data ...
```

Reply:
That DEBUG line isn't from the Messaging-Services, it's from the DSC's AbstractMessageService class directly. Needs to be removed there, will open an Issue in DSC repo. Can't change the DEBUG line here.

Reply:
https://github.com/International-Data-Spaces-Association/DataspaceConnector/blob/3381f147fd7935a1bc05dc5933b45d2ada639eda/src/main/java/io/dataspaceconnector/service/message/builder/type/base/AbstractMessageService.java#L127